lib/landrush/cap/linux/read_host_visible_ip_address.rb

```diff
@@ -39,7 +39,7 @@ def self.read_host_visible_ip_address(machine)
         end
 
         def self.command
-          %Q(hostname -I)
+          %Q(hostname -I 2>/dev/null || /sbin/ip ad | sed -ne 's/^\\W*inet \\([0-9.]*\\).* global .*$/\\1/p' 2>/dev/null)
```

Comment:
You cannot really assume _ip_ is installed. This was a long standing discussion. It is not. The whole of _read_host_visible_ip_address_ will be changed by pull request #193. We will be introducing a new guest capability which will copy a binary onto the guest which will determine the available interfaces.